api/src/test/java/uk/gov/dhsc/htbhf/claimant/utilities/DatabasePopulator.java

```diff
@@ -97,19 +117,53 @@ void generateClaimsAndPaymentCycles() {
             EligibilityStatus paymentEligibility = PAYMENT_ELIGIBILITY_STATUS.get(status);
             String ninoPrefix = NINO_PREFIX.get(status);
             Query query = entityManager.createNativeQuery("select count(*) from claimant where left(nino , 2) = '" + ninoPrefix + "'");
-            int ninoStartIndex = ((Number)query.getSingleResult()).intValue() + 1;
+            int ninoStartIndex = ((Number) query.getSingleResult()).intValue() + 10001; // ensure we have a non-zero number in the first 2 digits
             log.error("Creating {} claims in status {}, starting from {}", claims, status, ninoStartIndex);
+            List<Callable<Long>> jobs = new ArrayList<>(claims);
             for (int i = 0; i < claims; i++) {
```

Comment:
If you create the job outside the loop, you can use the Collections.nCopies(claims, job) to create the list.

Reply:
I don't think I can - the variable values change each time